Patch the specs file to no longer depend on a second libstdc++ build against
picolibc.

Currently libstdc++ is build as part of the g++-cross-embedded packages against
newlib and there is no second package. Using libstdc++ functions in conjunction
with picolibc may not work until there is a second libstdc++ build against
picolibc. But without this patch, even code that does not link against libstdc++
will fail to build.

--- a/meson.build	2026-07-31 05:29:08.000000000 +0200
+++ b/meson.build	2026-08-10 07:09:06.147284968 +0200
@@ -2081,13 +2081,6 @@ else
 
   gcc_dumpmachine = run_command(cc.cmd_array() + ['-dumpmachine'], check : true).stdout().split('\n')[0]
 
-  cxxconfig_path_postfix = '..' / gcc_dumpmachine / 'include' / 'c++' / '%(version)' / gcc_dumpmachine / '%M'
-  cxxconfig_isystem_prefix = isystem_prefix / cxxconfig_path_postfix
-  cxxconfig_isystem_buildtype = isystem_buildtype / cxxconfig_path_postfix
-  cxxconfig_isystem_gen = isystem_gen / cxxconfig_path_postfix
-
-  specs_isystem = specs_isystem + ' -isystem ' + specs_option_format.format(cxxconfig_isystem_prefix, cxxconfig_isystem_buildtype, cxxconfig_isystem_gen)
-
   #
   # Build the non-multilib -L value
   #
